spec/views/shared/dialogs/_dialog_field_drop_down_list.html.haml_spec.rb

```diff
@@ -0,0 +1,18 @@
+describe "shared/dialogs/dialog_field_drop_down_list.html.haml" do
+  context 'display field value' do
+    before do
+      @edit = false
+      attributes = {:values => [[nil, "<Choose>"], %w(Value1 Desc1), %w(Value2 Desc2)]}
+      value = 'Value2'
+      @field = DialogFieldDropDownList.new(:id => 1,
+                                           :name => 'Options List Field',
+                                           :type => 'DialogFieldDropDownList',
+                                           :attributes => attributes, :value => value)
+    end
+
+    it 'shows the display value for the dropdown field' do
+      render :partial => "/shared/dialogs/dialog_field_drop_down_list.html.haml", :locals => {:edit => @edit, :field => @field}
+      expect(rendered).to eq("Desc2\n")
+    end
```

Comment:
Can we add more tests here around all of the different branches the UI is taking? Right now, the test doesn't really explain why we have to use `try` in a few places and when things are nil. Would feel much more comfortable if all of the cases are covered so that when/if it actually happens, it doesn't just crash.

However, I think ultimately I would prefer if this functionality were in a helper so we didn't have to check the rendered data, and the helper could potentially be used elsewhere.